### ISSUE TITLE
UMD module name

### ISSUE THREAD

**Issue by Micene09** [apexcharts/apexcharts.js#210]
Hi, as i see in dist file, the UMD builded code is:
```javascript
(function webpackUniversalModuleDefinition(root, factory) {
	if(typeof exports === 'object' && typeof module === 'object')
		module.exports = factory();
	else if(typeof define === 'function' && define.amd)
		define("ApexCharts", [], factory);
	else if(typeof exports === 'object')
		exports["ApexCharts"] = factory();
	else
		root["ApexCharts"] = factory();
})(this, function() {
```

I'm currently using require.js to import this module.
Usually in AMD projects, using the module name in 3rd party dependency give some problems during imports (undefined references), can you make it anonymous as the example below please?
```javascript
 define([], factory);
```

**Comment by junedchhipa** [comment]
I have less knowledge about umd/amd, but you can try editing the webpack configuration here - https://github.com/apexcharts/apexcharts.js/blob/master/webpack.config.js

You can change `libraryTarget: 'umd'` to `libraryTarget: 'amd'`
Create a build - and see if it works correctly. 
I would accept a PR if this isn't a breaking change.

Or - as an option - I can provide a separate file specifically for amd `apexcharts.amd.min.js` after testing it.

**Comment by Micene09** [comment]
A separate file should be the safer and fastest solution, if you feel confident too i think you can go and provide a separate amd build (normal + minified).

**Comment by junedchhipa** [comment]
Cool, I'll try that

**Comment by Micene09** [comment]
According to the webpack's doc, you can set the [umdNamedDefine ](https://webpack.js.org/configuration/output/#output-umdnameddefine)to false.

Just apply this and you don't need to provide a separate build, i tested it and it's working as expected:
```javascript
(function webpackUniversalModuleDefinition(root, factory) {
	if(typeof exports === 'object' && typeof module === 'object')
		module.exports = factory();
	else if(typeof define === 'function' && define.amd)
		define([], factory); // << anonymous define finally, we got it!
	else if(typeof exports === 'object')
		exports["ApexCharts"] = factory();
	else
		root["ApexCharts"] = factory();
})(this, function() {
```

**Comment by junedchhipa** [comment]
Alright, the change will be released in 2.2.4
